.serena/memories/suggested_commands.md

```
@@ -0,0 +1,10 @@
+# Suggested Commands
+- `cd src && go run cmd/api/main.go` — run the HTTP API locally (requires CGO enabled for SQLite).
+- `cd src && CGO_ENABLED=1 go build -o ../build/digests-api-linux-amd64 ./cmd/api` — build release binary to `build/`.
+- `cd src && go test ./...` — run Go tests across all packages.
+- `cd src && go tool golangci-lint run --config ../.golangci.yml ./...` — lint using repository configuration (install via `go get -tool` if missing).
+- `cd src && go fmt ./...` — format Go code.
+- `cd src && make run|build|test|lint` equivalents — Makefile shortcuts for running, building, and orchestrating scripts.
+- `./docker-entrypoint.sh` or `docker build -t digests-api .` — container build/run flow when needed.
+- `scripts/logs/logs-pretty.sh` — tail formatted logs; `scripts/database/db-*.sh` — inspect SQLite tables.
+- `git submodule update --init --recursive` — ensure Hermes submodule is synced.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)